scout/server/blueprints/variant/templates/variant/utils.html

```diff
@@ -37,14 +37,14 @@ <h6><strong>Associated phenotype</strong></h6>
                   <h6>Diagnosis phenotype (OMIM):</h6>
                   <select multiple="multiple" name="omim_select" id="omim_select" class="selectpicker">
                     {% for omim_term in case.diagnosis_phenotypes %}
-                      {% if 'disease_nr' in omim_term %}
+                      {% if omim_term is number %} <!-- Old way of saving OMIM terms in cases: list of diagnosis numbers -->
```

Comment:
Same here in a way, it would be cleaner to have this dealt with in the controller, e.g. with pydantic 😛, but this was already in the template, should work, and fixes the bug.